File: 0058-length-of-last-word/58.length-of-last-word.cs
/*
 * @lc app=leetcode id=58 lang=csharp
 *
 * [58] Length of Last Word
 */

// @lc code=start
public class Solution {
    public int LengthOfLastWord(string s) {
        string[] words = s.Trim().Split(' ');

        return words[words.Length-1].Length;
    }
}
// @lc code=end

File: 0058-length-of-last-word/58.length-of-last-word.py
#
# @lc app=leetcode id=58 lang=python3
#
# [58] Length of Last Word
#

# @lc code=start
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        word_list = [word for word in s.split(" ") if word != '']

        return len(word_list[-1])
        
# @lc code=end

